app/models/user.rb

```diff
@@ -24,14 +27,16 @@
 #
 
 class User < ApplicationRecord
+  MAX = 250
```

Comment:
MAX variable is not meaningful ... maybe MAX_INPUT_LENGTH would be better, but actually I liked previous version like max: 255 more, since I don't need to spend time and check what is the actualy value.

revert